src/test/java/back/petionary/account/entity/AccountTest.java

```diff
@@ -0,0 +1,171 @@
+
+    @Test
+    @DisplayName("유저의 정보 중 이메일이 null일 경우 예외가 발생한다.")
+    void isNullEmail() {
+        //given
+        String email = null;
+        String name = "테스트";
+        LocalDate birth = LocalDate.of(1998,10,15);
+        String zipcode = "1111";
+        String detail = "상세주소";
+        String phone = "01011111111";
+        Address address = new Address(zipcode, detail);
+
+        //when && then
+        CommonException commonException = assertThrows(CommonException.class,
+            () -> new Account(email, phone, name, birth, address));
+        assertThat(commonException.getErrorCode().getStatus()).isEqualTo(HttpStatus.BAD_REQUEST);
+    }
+    @Test
+    @DisplayName("유저의 정보 중 이름이 null일 경우 예외가 발생한다.")
+    void isNullName() {
+        //given
+        String email = "test@test.com";
+        String name = null;
+        LocalDate birth = LocalDate.of(1998,10,15);
+        String zipcode = "1111";
+        String detail = "상세주소";
+        String phone = "01011111111";
+        Address address = new Address(zipcode, detail);
+
+        //when && then
+        CommonException commonException = assertThrows(CommonException.class,
+            () -> new Account(email, phone, name, birth, address));
+        assertThat(commonException.getErrorCode().getStatus()).isEqualTo(HttpStatus.BAD_REQUEST);
+    }
+    @Test
+    @DisplayName("유저의 정보 중 생년월일이 null일 경우 예외가 발생한다.")
+    void isNullBirth() {
+        //given
+        String email = "test@test.com";
+        String name = "test";
+        LocalDate birth = null;
+        String zipcode = "1111";
+        String detail = "상세주소";
+        String phone = "01011111111";
+        Address address = new Address(zipcode, detail);
+
+        //when && then
+        CommonException commonException = assertThrows(CommonException.class,
+            () -> new Account(email, phone, name, birth, address));
+        assertThat(commonException.getErrorCode().getStatus()).isEqualTo(HttpStatus.BAD_REQUEST);
+    }
+
+    @Test
+    @DisplayName("유저의 정보 중 핸드폰번호가 null일 경우 예외가 발생한다.")
+    void isNullPhone() {
+        //given
+        String email = "test@test.com";
+        String name = "테스트";
+        LocalDate birth = LocalDate.of(1998,10,15);
+        String zipcode = "1111";
+        String detail = "상세주소";
+        String phone = null;
+        Address address = new Address(zipcode, detail);
+
+        //when && then
+        CommonException commonException = assertThrows(CommonException.class,
+            () -> new Account(email, phone, name, birth, address));
+        assertThat(commonException.getErrorCode().getStatus()).isEqualTo(HttpStatus.BAD_REQUEST);
+
+    }
+
+    @Test
+    @DisplayName("유저의 정보 중 상세주소가 null일 경우 예외가 발생한다.")
+    void isNullDetail() {
+        //given
+        String zipcode = "1111";
+        String detail = null;
+
+        //when && then
+        CommonException commonException = assertThrows(CommonException.class,
+            () ->new Address(zipcode, detail));
```

Comment:
```suggestion
            () -> new Address(zipcode, detail));
```

이런 부분은 리포맷팅 돌렸으면 나오지 않을 부분이라 아쉽네요